libs/sdk-ui-filters/src/AttributeFilter@next/Components/DropdownButton/AttributeFilterDropdownButton.tsx

```diff
@@ -70,10 +85,12 @@ export const AttributeFilterDropdownButton: React.VFC<IAttributeFilterDropdownBu
                     "gd-is-filtering": isFiltering,
                     "gd-is-active": isOpen,
                     "gd-is-loaded": isLoaded,
+                    "gd-is-draggable": isDraggable,
                 },
             )}
             onClick={onClick}
         >
+            {icon && <div className="gd-attribute-filter-dropdown-button-icon__next"> {icon}</div>}
```

Comment:
Is the space before the `{icon}` intentional?

Reply:
fixed